### PR TITLE
chore: [IOBP-92] Update IDPay API specifications

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR updated the IDPay API specifications url in package.json

## How to test
Tests should solve correctly

### CODE REVIEW

File: ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx [hunk 1]
```diff
@@ -105,18 +98,31 @@ const OperationAmountOrLabel = ({ operation }: OperationComponentProps) => {
...
       return (
-        <H4>{`-${formatNumberAmount(
-          Math.abs(operation.accrued),
-          false
-        )} €`}</H4>
+        <H4>
+          {pipe(
+            operation.accrued,
+            O.fromNullable,
+            O.map(accrued => Math.abs(accrued)),
```
Comment (forrest57) [inline ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx:107]:
```suggestion
            O.map(Math.abs),
```

File: ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx [hunk 2]
```diff
@@ -105,18 +98,31 @@ const OperationAmountOrLabel = ({ operation }: OperationComponentProps) => {
...
       return (
-        <H4>{`+${formatNumberAmount(
-          Math.abs(operation.accrued),
-          false
-        )} €`}</H4>
+        <H4>
+          {pipe(
+            operation.accrued,
+            O.fromNullable,
+            O.map(accrued => Math.abs(accrued)),
```
Comment (forrest57) [inline ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx:120]:
```suggestion
            O.map(Math.abs),
```

File: ts/features/idpay/initiative/timeline/components/TimelineTransactionDetailsComponent.tsx [hunk 3]
```diff
@@ -47,6 +47,18 @@ const TimelineTransactionDetailsComponent = (props: Props) => {
...
 
+  const idTrxIssuer = transaction.idTrxIssuer || "";
+  const idTrxAcquirer = transaction.idTrxAcquirer || "";
+
+  const formattedAmount = pipe(
+    transaction.amount,
+    O.fromNullable,
+    O.map(amount => formatNumberAmount(amount, true)),
+    O.getOrElse(() => "-")
+  );
```
Comment (forrest57) [inline ts/features/idpay/initiative/timeline/components/TimelineTransactionDetailsComponent.tsx:58]: what about a `formatNumberCurrencyOrDefaultRightSign` function? or maybe a small refactor of the already existing function in order to pass an optional boolean parameter

Comment (mastro993) [reply ts/features/idpay/initiative/timeline/components/TimelineTransactionDetailsComponent.tsx:58]: It's only used here, IMHO there is no need to generalize its usage for now.

**Review by forrest57 — APPROVED** [review]

**Pull request merged**